### PR TITLE
Styling improvements

### PR DESCRIPTION
**Pull request by kierangillen**
- https://artsyproduct.atlassian.net/browse/PURCHASE-1235
- https://artsyproduct.atlassian.net/browse/PURCHASE-1227

- Adds checks for sections before rendering the sections in `FlatList`

- Fixes name and follow button

Screen Shot 2019-07-08 at 6 11 28 PM

- Fixes Read More extra space

Screen Shot 2019-07-08 at 6 11 39 PM

#trivial

### CODE REVIEW

File: src/lib/Scenes/Artwork/Components/ArtworkHeader.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ export class ArtworkHeader extends React.Component<ArtworkHeaderProps> {
     const { artwork } = this.props
     return (
       <Box>
+        <Spacer mb={2} />
```
Comment (kierangillen) [inline src/lib/Scenes/Artwork/Components/ArtworkHeader.tsx]: Adds 20px above image in header

File: src/lib/Scenes/Artwork/Components/ArtworkTombstone.tsx [hunk 2]
```diff
@@ -39,10 +39,13 @@ export class ArtworkTombstone extends React.Component<ArtworkTombstoneProps, Art
   renderSingleArtist(artist: Artist) {
     return (
       <React.Fragment>
-        <Flex flexDirection="row">
```
Comment (kierangillen) [inline src/lib/Scenes/Artwork/Components/ArtworkTombstone.tsx]: Removes flex component and renders all the text in a text element for proper alignment.

File: src/lib/Scenes/Artwork/Components/FollowArtistButton.tsx [hunk 3]
```diff
@@ -44,16 +44,11 @@ export class FollowArtistButton extends React.Component<Props> {
   render() {
     const followButtonText = this.props.artist.is_followed ? "Following" : "Follow"
     return (
-      <>
-        <Sans color="black60" size="6" mx={1}>
-          &middot;
```
Comment (ds300) [inline src/lib/Scenes/Artwork/Components/FollowArtistButton.tsx]: good idea moving this out to the tombstone

File: src/lib/Scenes/Artwork/Components/ArtworkAvailability.tsx [hunk 4]
```diff
@@ -12,7 +12,7 @@ export class ArtworkAvailability extends React.Component<ArtworkAvailabilityProp
   render() {
     return (
       <Box>
-        <Sans size="4t" weight="medium">
+        <Sans size="3t" weight="medium">
```
Comment (sweir27) [inline src/lib/Scenes/Artwork/Components/ArtworkAvailability.tsx]: Not sure if I'm looking at something outdated, but my zeplin spec has this being `4t`.
image

Comment (kierangillen) [reply src/lib/Scenes/Artwork/Components/ArtworkAvailability.tsx]: Ah good catch thanks

**Pull request merged**